cluster/common.sh

```diff
@@ -1132,10 +1132,8 @@ function registry-authentication() {
 function create-and-upload-etcd-image() {
 
   ETCD_IMAGE_REGISTRY="${ETCD_IMAGE_REGISTRY:-${CONTAINER_REGISTRY}/${PROJECT}}"
-
   # Build+push the image through makefile.
   MAKE_DIR="${KUBE_ROOT}/cluster/images/etcd"
-
```

Comment:
this commit already included in master.

Reply:
Can you specify the commit comment?

Reply:
included in this PR: https://github.com/CentaurusInfra/arktos/pull/796